update.go

```diff
@@ -36,6 +38,13 @@ func (b *UpdateStmt) Build(d Dialect, buf Buffer) error {
 		return ErrColumnNotSpecified
 	}
 
+	for _, comment := range b.Comments {
+		comment = strings.TrimSpace(comment)
+		buf.WriteString("-- ")
+		buf.WriteString(comment)
```

Comment:
I guess that depends on precedence of `--` and `/*!..*/` commented code. i.e. is this a regular comment or an executable comment:
```
-- /*! MySQL-specific code */

```

Reply:
Fortunately, MySQL treats the rest of the line after `-- ` as a comment, even if it has a `/*` or `/*!`

However, I'm concerned about an element in the `Comments` slice that contains a newline, since that seems like it would add executable code to the statement.

Reply:
It seems like there are a few solutions to this:
1. Stripping/swapping out new lines, which may lead to surprising behavior
2. Splitting the text on newline  and creating a new comment for each segment.
3. Use multiline comments /* ... */
4. Or just issuing a warning in the log/documentation

3 seems like the best/most-performant

Reply:
Option 3 sounds good. It appears that this style of multiline comment is essentially standard SQL. 

https://jakewheat.github.io/sql-overview/sql-2003-foundation-grammar.html

https://stackoverflow.com/a/728326